src/utils/rlp.cairo

```diff
@@ -140,4 +141,86 @@ namespace RLP {
             return (rlp_len=data_len + 1);
         }
     }
+
+    // @notice encodes felt into an rlp element
+    // @param element The felt that is encoded into rlp
+    // @param rlp_len The length of the rlp array
+    // @param rlp The pointer receiving the rlp encoded felt
+    // @return rlp_len The length of the encoded felt in bytes
+    func encode_element_felt{
```

Comment:
motivation: to encode the salt/nonce value in rlp element (which has distinct logic from an rlp list).

see: https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/

Reply:
could be named just `encode_felt`?